package.jsox

```diff
@@ -16,17 +16,15 @@
     "@std/esm": "cjs",
     module: "lib/jsox.mjs",
     main: "lib/jsox.js",
-    browser: "lib/jsox.es6.js.gz",
```

Comment:
This `lib/jsox.es6.js.gz` file generated by `build` script is actually an ESM output, we can just use the mjs entry instead.